tfx/examples/cifar10/distributed/cifar10_utils_native_keras.py

```diff
@@ -0,0 +1,430 @@
+# Lint as: python2, python3
```

Comment:
Is there a way we can do this without duplicating the util module?

Reply:
I think there are around 4 places and about 30 lines total that are different from the original util module, so it's a non-trivial change for the users to make. That being said, it should be possible to incorporate all of these changes in the README.